docs/samples/custom/torchserve/README.md

```diff
@@ -0,0 +1,83 @@
+# Predict on a InferenceService using a Custom Torchserve Image
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/#install-kfserving).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
```

Comment:
Can you mention that this example only works with v1beta1/KFS 0.5?

Reply:
Added.